README.rst

```diff
@@ -2,7 +2,7 @@
 
 |build-status| |coverage| |license| |wheel| |pyversion| |pyimp| |ocbackerbadge| |ocsponsorbadge|
 
-:Version: 4.4.0rc3 (cliffs)
+:Version: 5.0.0a1 (cliffs)
```

Comment:
Don't touch this.
Bumpversion is responsible for bumping the version.

Reply:
Oh OK. I never used that tool so need your help on that :D also for the next releases willing to use something like bumpversion